Antonio_Ramos_Documentation

```diff
@@ -381,3 +381,80 @@ Tab2 = Tab2.astype(convert_dict)
 
 - Problems left to resolve - 1. no cluster file is being generated even after 499 simulations 2. the find and replace done on the excel file needs to be automated in python 3. the code that produces the labs data file is not producing files which project correctly 
 
+8/8/2020
+
+- installying arcpy to python envinronment "programTrial" using following command in anaconda command prompt "conda install -c esri arcpy" 
```

Comment:
Please add some context here for why you're doing this, why you need arcpy, what program you're working on updating

Reply:
Will do. Arcpy is a package giving us the recalculate command.

Reply:
Now I feel like I should have been able to figure it out. But I was trying to figure out what "packed" meant in this context and got blinded by it